scripts/filter_language_issues.py

```
@@ -0,0 +1,181 @@
+    if rule_id == "WHITESPACE_RULE":
+        return True
+
+    # Ignore comma/parenthesis whitespace in code blocks
+    if rule_id == "COMMA_PARENTHESIS_WHITESPACE":
+        return True
+
+    # Ignore very short text snippets (likely code)
+    if len(text.strip()) <= 3:
+        return True
+
+    return False
```

Ruff (0.12.2)

Return the condition `len(text.strip()) <= 3` directly

Replace with `return len(text.strip()) <= 3`

(SIM103)